### DISCUSSION TITLE
[RFC] Settings icon in home screen

### DISCUSSION THREAD

**Discussion by arky** [digidem/mapeo-mobile#812] [Ideas]
To get to settings screen, you'll have to click 'observations icon' and then select settings.  The better approach is to provide a settings icon directly on the home screen. 

Am seeking your comments on what is best place for such an settings icon without hampering the current user experience. Please let me know your ideas in the comments. 

My proposal is to place a setting gear icon in the panel that displays GPS fix on the top middle of the home screen.

**Comment by achou11** [comment]
hi @arky, we've got some designs that address this and will be directly accessible from the home screen once implemented! it'll most likely be a settings icon that's located where the observations icon currently is but still working through a few things

**Comment by arky** [comment]
Thanks @achou11 Please do share any mock-ups you have. I'll be able give some feedback and also get them validated with quick user testing internally.
